public/app/plugins/datasource/elasticsearch/dataquery.cue

```diff
@@ -49,9 +49,9 @@ composableKinds: DataQuery: {
 						#BucketAggregationType: "terms" | "filters" | "geohash_grid" | "date_histogram" | "histogram" | "nested" @cuetsy(kind="type")
 
 						#BaseBucketAggregation: {
-							id:   string
-							type: #BucketAggregationType
-							settings?: {...}
+							id:        string
+							type:      #BucketAggregationType
+							settings?: _
```

Comment:
This is strange, if `settings` is defined as `_` instead of `{...}`, the `settings` field gets correctly added to all the child types. I've opened https://github.com/grafana/schematization-and-as-code-project/issues/78 to track this.